lib/api/src/grpc/qdrant.rs

```diff
@@ -2941,6 +2941,30 @@ pub mod points_server {
     }
 }
 #[derive(Clone, PartialEq, ::prost::Message)]
+pub struct SyncPoints {
+    /// name of the collection
+    #[prost(string, tag="1")]
+    pub collection_name: ::prost::alloc::string::String,
+    /// Wait until the changes have been applied?
+    #[prost(bool, optional, tag="2")]
+    pub wait: ::core::option::Option<bool>,
+    #[prost(message, repeated, tag="3")]
+    pub points: ::prost::alloc::vec::Vec<PointStruct>,
+    /// Start of the sync range
+    #[prost(message, optional, tag="4")]
+    pub from_id: ::core::option::Option<PointId>,
+    /// End of the sync range
+    #[prost(message, optional, tag="5")]
+    pub to_id: ::core::option::Option<PointId>,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct SyncPointsInternal {
+    #[prost(message, optional, tag="1")]
+    pub sync_points: ::core::option::Option<SyncPoints>,
```

Comment:
I am a bit confused, why is `sync_points` optional if it is not marked as so in `proto` file.

Reply:
Ah I see, it just was strange that `prost` does this only for user types and for scalar types it will use default value if it is missing. But I found it in docs https://github.com/tokio-rs/prost#field-modifiers